app/controllers/application.js

```diff
@@ -1,12 +1,14 @@
 import Controller from '@ember/controller';
 import { inject as service } from '@ember/service';
-import { LOGOUT_URL, MAIN_SITE_URL } from '../constants/navbar';
+import { LOGOUT_URL, MAIN_SITE_URL } from '../constants/url';
 import { action } from '@ember/object';
 import { GITHUB_URL } from '../constants/url';
+import ENV from 'website-my/config/environment'; // remove this when new flow goes live
 
 export default class ApplicationController extends Controller {
   @service router;
   GITHUB_URL = GITHUB_URL;
+  BASE_API_URL = ENV.BASE_API_URL;
```

Comment:
Can we please use this `URL` while making a fetch call and remove the `LOGOUT_URL` from here?
![image](https://user-images.githubusercontent.com/78433013/216759061-ffbbd259-d9c7-4850-9910-21486f547e95.png)